pkg/base1/test-echo.js

```diff
@@ -83,7 +83,7 @@ QUnit.test("fence", async assert => {
     const done = assert.async();
 
     // This is implemented in the C bridge, but not in Python.
-    if (await QUnit.mock_info("bridge") == "cockpit-bridge.pyz") {
+    if (await QUnit.mock_info("bridge") == "python3") {
```

Comment:
Can we keep the unit tests work with either? Maybe in the simplest case with `.includes("py")`?

Reply:
Sure.  I actually think maybe we should add a helper function for this, then.

Another option would be to export that as a boolean flag and move the logic to `./test-server` itself, closer to the implementation.

Reply:
Done.